data/wanderer/wanderers middle.txt

```diff
@@ -3397,6 +3397,7 @@ mission "Wanderers: Sestor: Final"
 		event "wanderers: exiles have drones" 20
 		event "wanderers: sabira eseskrai colony" 30
 		event "wanderers: sestor eliminated" 40
+		event "wanderers taking jobs from kor efreti" 5
```

Comment:
The fleets don't add anything hugely substantial, and they're a rare sight as-is. That said, I think it would feel inconsistent for old saves to not have them while new saves do. As long as the amount of lines it would add (less than 10) isn't a thing to be concerned about, I'll go ahead and add one in next to the existing patch mission near the bottom of `wanderers middle.txt`.